# HG changeset patch
# User Rik <rik@octave.org>
# Date 1464116829 25200
#      Tue May 24 12:07:09 2016 -0700
# Node ID cf3941f99c3594793feccad580e2f818ef11bacb
# Parent  bd51eabb38ed80c5f69d16bd8cd00b2486f820c3
doc: Remove incorrect text regarding 'tex' interpreter.

* plot.txi: Remove text saying that results are not available
in printed output from OpenGL toolkits.

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -631,20 +631,16 @@ rendering occurs---the displayed text is
 Currently, the @qcode{"latex"} interpreter is not implemented and is equivalent
 to @qcode{"none"}.
 
 The @qcode{"tex"} option implements a subset of @TeX{} functionality when
 rendering text.  This allows the insertion of special glyphs such as Greek
 characters or mathematical symbols.  Special characters are inserted by using
 a backslash (\) character followed by a code, as shown in @ref{tab:extended}.
 
-Note that for on-screen display the interpreter property is honored by all
-graphics toolkits.  However for printing, @strong{only} the @qcode{"gnuplot"}
-toolkit renders @TeX{} instructions.
-
 Besides special glyphs, the formatting of the text can be changed within the
 string by using the codes
 
 @multitable @columnfractions .2 .2 .6 .2
 @item @tab \bf @tab Bold font @tab
 @item @tab \it @tab Italic font @tab
 @item @tab \sl @tab Oblique Font @tab
 @item @tab \rm @tab Normal font @tab
